src/components/DetailArticle.jsx

```diff
@@ -1,25 +1,47 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect, useContext } from "react";
 import { useNavigate, useParams } from "react-router-dom";
 import {
   fetchArticleById,
   fetchComments,
+  fetchDeleteComment,
   fetchVotesUpdate,
 } from "../Utils/api";
 import { FaArrowAltCircleUp, FaArrowAltCircleDown } from "react-icons/fa";
 
 import "./DetailArticle.css";
 import { AddComment } from "./AddComment";
+import UserContext from "../contexts/UserContext";
 
 export const DetailArticle = () => {
   const [comments, setComments] = useState([]);
   const [article, setArticle] = useState(null);
   const [votes, setVotes] = useState(0);
   const [error, setError] = useState(null);
   const [loading, setLoading] = useState(true);
+  const [deletingComment, setDeletingComment] = useState(false);
+  const { currentUser } = useContext(UserContext);
 
+  
   const { article_id } = useParams();
   const navigate = useNavigate();
 
+
+  const handleDeleteComment = (comment_id) => {
+  console.log(comment_id)
+    fetchDeleteComment(comment_id)
+      .then(() => {
+        setLoading(false);
+        window.location.reload()
+      })
+      .catch((error) => {
+        setError(error.response.msg);
```

Comment:
When there's an error deleting the comment (like if the internet cuts out) this error component doesn't seem to be appearing, the comment doesn't delete which is a good visual indication something's wrong, but it would be good if this error component also provided a message!